webapp/javascript/redux/reducers/user.ts

```diff
@@ -34,6 +34,10 @@ export const loadCurrentUser = createAsyncThunk(
     if ('code' in res.error && res.error.code === 404) {
       return Promise.resolve({ id: 0, role: 'anonymous' });
     }
+    // Suppress 401 error on login screen
+    if ('code' in res.error && window?.location?.pathname === '/login') {
```

Comment:
if a user uses baseURL I think this might be something other than login, right? e.g if baseURL = `/pyroscope` this page will be at `/pyroscope/login`

Reply:
Noted. 

Yeah that's bascially why we need e2e tests